rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/expression/ExpressionGenerator.xtend

```diff
@@ -1015,14 +1007,9 @@ class ExpressionGenerator extends RosettaExpressionSwitch<JavaStatementBuilder,
 				val attribute = rObjectFactory.buildRAttribute(s)
 				// Data attributes can only be called if there is an implicit variable present.
 				val implicitType = typeProvider.typeOfImplicitVariable(expr)
-				val implicitFeatures = implicitType.allFeatures(expr)
+				val implicitFeatures = implicitType.RType.allFeatures(expr)
```

Comment:
Why take `RType` here? Does this not break expressions such as `stringsWithScheme extract scheme`?